pineappl/src/subgrid.rs

```diff
@@ -115,6 +124,15 @@ pub trait Subgrid {
 
     /// Return the static (single) scale, if this subgrid has one.
     fn static_scale(&self) -> Option<Mu2>;
+
+    /// Return the array
```

Comment:
I know the name is good enough on its own, but like this, this docstring is not very informative...
```suggestion
    /// Fill a dense array with subgrid content
```